doc/dev_guide/lazy_computations.rst

```diff
@@ -46,6 +46,20 @@ cases for lazy signals will have to be written. If a function operates on
 arbitrary-sized arrays and the shape of the output can be known before calling,
 ``da.map_blocks`` and ``da.map_overlap`` are efficient and flexible.
 
+For more recent versions of dask (dask>2021.11) when using hyperspy in a jupyter
+notebook a helpful html representation is available.
+    .. code-block::python
```

Comment:
This example needs to use a lazy signal and it doesn't appear in the document, maybe because of the indentation?